src/dunes/__main__.py

```diff
@@ -80,7 +80,7 @@ def load_all_modules_from_dir(plugin_dir):
 
     handle_simple_args()
 
-    dune_sys = dune(args)
+    dune_sys = dunes(args)
```

Comment:
Consider renaming `dune_sys` to `dunes_sys`? 
```suggestion
    dunes_sys = dunes(args)
```

Reply:
renamed